pkgs/by-name/ld/ldproxy/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, rustPlatform, fetchFromGitHub, pkg-config, udev, stdenv }:
+
+# For more information on using `buildRustPackage` please take a look at:
+# https://nixos.org/manual/nixpkgs/stable/#compiling-rust-applications-with-cargo
```

Comment:
This is too obvious.

Reply:
Potentially fixed in 2668391e4ad8d4737bc4327fab3031ee2f27e6bf